Habit-Tracker-JVR-Hannes/Habit-Tracker/Program.cs

```diff
@@ -0,0 +1,346 @@
+                tableCmd.Parameters.AddWithValue("@habitType", habitType);
+
+                List<Habit> tableData = new();
+                SqliteDataReader reader = tableCmd.ExecuteReader();
+
+                if (reader.HasRows)
+                {
+                    while (reader.Read())
+                    {
+                        tableData.Add(
+                        new Habit
+                        {
+                            Id = reader.GetInt32(0),
+                            HabitType = reader.GetString(1),
+                            Date = DateTime.ParseExact(reader.GetString(2), "dd-MM-yyyy", new CultureInfo("en-US")),
+                            Quantity = reader.GetInt32(3)
+                        });
+                    }
+                }
+                else
+                {
+                    Console.WriteLine("No records found for this habit.");
+                }
+
+                connection.Close();
+
+                Console.WriteLine("+++++++++++++++++++++++++++++++++++++++++\n");
+                foreach (var habit in tableData)
+                {
+                    Console.WriteLine($"{habit.Id} - {habit.HabitType} - {habit.Date.ToString("dd-MM-yyyy")} - Quantity: {habit.Quantity}");
+                }
+                Console.WriteLine("+++++++++++++++++++++++++++++++++++++++++\n");
+            }
+        }
+
+        private static void Update()
+        {
+            Console.Clear();
+            GetAllRecords();
+
+            var recordId = GetNumberInput("\n\nPlease type Id of the record you would like to update. Type 0 to return to main menu. \n\n");
+
+            if (recordId == 0)
+                return;
+
+            using (var connection = new SqliteConnection(connectionString))
+            {
+                connection.Open();
+
+                var checkCmd = connection.CreateCommand();
+                checkCmd.CommandText = $"SELECT EXISTS(SELECT 1 FROM habits WHERE Id = {recordId})";
+                int checkQuery = Convert.ToInt32(checkCmd.ExecuteScalar());
+
+                if (checkQuery == 0)
+                {
+                    Console.WriteLine($"\n\nRecord with Id {recordId} doesn't exist.\n\n");
+                    return;
+                }
+
+                string date = GetDateInput();
+
+                int quantity = GetNumberInput("\n\nPlease insert number of measure of your choice " +
+                    "(no decimals allowed)\n\n");
+
+                var tableCmd = connection.CreateCommand();
+                tableCmd.CommandText = $"UPDATE habits SET date = @date, quantity = @quantity WHERE Id = @recordId";
+                tableCmd.Parameters.AddWithValue("@date", date);
+                tableCmd.Parameters.AddWithValue("@quantity", quantity);
+                tableCmd.Parameters.AddWithValue("@recordId", recordId);
+
+                tableCmd.ExecuteNonQuery();
+
+                connection.Close();
+            }
+
```

Comment:
Remember you Spacing [code-conventions](https://thecsharpacademy.com/article/58/code-conventions)

- Remove empty lines after opening curly brackets and before closing curly brackets.

![image](https://github.com/user-attachments/assets/a8385d03-6bc4-46c9-8f3c-519f3e1bf9f4)